pyscf/cc/__init__.py

```diff
@@ -183,29 +183,27 @@ def RQCISD(mf, frozen=None, mo_coeff=None, mo_occ=None):
 RQCISD.__doc__ = qcisd.QCISD.__doc__
 
 
-def FNOCCSD(mf, thresh=1e-6, pct_occ=None, nvir_act=None, frozen=None):
+def FNOCCSD(mf, thresh=1e-6, pct_occ=None, pvir_act=None, nvir_act=None, frozen=None):
     """Frozen natural orbital CCSD
 
     Attributes:
         thresh : float
             Threshold on NO occupation numbers. Default is 1e-6 (very conservative).
         pct_occ : float
             Percentage of total occupation number. Default is None. If present, overrides `thresh`.
+        pvir_act : float
+            Percentage of total number of virtuals (ceiling is taken to get an integer number).
```

Comment:
Is this "Percentage of number of virtual NOs to keep" or something else? (I'm comparing this to the comment below "nvir_act").

Reply:
The "Percentage of total occupation number" is called `pct_occ`. `pvir_act` doesn't use that naming convention (no "pct") but follows `nvir_act` (which makes sense but might be a little bit inconsistent) <- minor comment.